agile_problem1.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Problem1 do
+	import Integer
+	
+	#Main public function that sums up the even numbers
+	def sum_even(), do: 0
+	def sum_even(list), do: do_sum_even(list, 0)
+	
+	
+	#Private functions ---------------------------------------
+	
+	defp do_sum_even([], acc), do: acc
+	
+	defp do_sum_even([head | tail], acc) do
+	    if is_even(head) do

```

Comment:
This is indeed a way to solve this problem, very c like, but it would have been nice to see the elixir way to do it.

This entire module can be replaced by a simple function that uses pipes and the Enum module.